tests/sqlparser_postgres.rs

```diff
@@ -5320,6 +5320,117 @@ fn parse_create_type_as_enum() {
+        TestCase {
+            sql: r#"ALTER TYPE public.my_type
+                        ADD VALUE 'label3.5' BEFORE 'label4'"#,
+            canonical: "ALTER TYPE public.my_type ADD VALUE 'label3.5' BEFORE 'label4'",
+            name: "public.my_type",
+            operation: AlterTypeOperation::AddValue {
+                if_not_exists: false,
+                value: Ident::with_quote('\'', "label3.5"),
+                position: AlterTypeOperationAddValuePosition::Before(Ident::with_quote(
+                    '\'', "label4",
+                )),
+            },
+        },
+        TestCase {
+            sql: r#"ALTER TYPE public.my_type
+                        ADD VALUE IF NOT EXISTS 'label3.5'
+                        AFTER 'label3'"#,
+            canonical:
+                "ALTER TYPE public.my_type ADD VALUE IF NOT EXISTS 'label3.5' AFTER 'label3'",
+            name: "public.my_type",
+            operation: AlterTypeOperation::AddValue {
+                if_not_exists: true,
+                value: Ident::with_quote('\'', "label3.5"),
+                position: AlterTypeOperationAddValuePosition::After(Ident::with_quote(
+                    '\'', "label3",
+                )),
+            },
+        },
+        TestCase {
+            sql: r#"ALTER TYPE public.my_type
+                        ADD VALUE 'label3.5'
+                        AFTER 'label3'"#,
+            canonical: "ALTER TYPE public.my_type ADD VALUE 'label3.5' AFTER 'label3'",
+            name: "public.my_type",
+            operation: AlterTypeOperation::AddValue {
+                if_not_exists: false,
+                value: Ident::with_quote('\'', "label3.5"),
+                position: AlterTypeOperationAddValuePosition::After(Ident::with_quote(
+                    '\'', "label3",
+                )),
+            },
+        },
+        TestCase {
+            sql: r#"ALTER TYPE public.my_type
+                        ADD VALUE IF NOT EXISTS 'label5'"#,
+            canonical: "ALTER TYPE public.my_type ADD VALUE IF NOT EXISTS 'label5'",
+            name: "public.my_type",
+            operation: AlterTypeOperation::AddValue {
+                if_not_exists: true,
+                value: Ident::with_quote('\'', "label5"),
+                position: AlterTypeOperationAddValuePosition::Default,
+            },
+        },
+        TestCase {
+            sql: r#"ALTER TYPE public.my_type
+                        ADD VALUE 'label5'"#,
+            canonical: "ALTER TYPE public.my_type ADD VALUE 'label5'",
+            name: "public.my_type",
+            operation: AlterTypeOperation::AddValue {
+                if_not_exists: false,
+                value: Ident::with_quote('\'', "label5"),
+                position: AlterTypeOperationAddValuePosition::Default,
+            },
+        },
+    ]
+    .into_iter()
+    .enumerate()
+    .for_each(|(index, tc)| {
+        let statement = pg_and_generic().one_statement_parses_to(tc.sql, tc.canonical);
```

Comment:
```suggestion
        let statement = pg_and_generic().verified_stmt(tc.sql);
```
can we use verified_stmt instead?